Zend/zend_vm_def.h

```diff
@@ -9236,6 +9236,17 @@ ZEND_VM_HANDLER(202, ZEND_CALLABLE_CONVERT, UNUSED, UNUSED)
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(203, ZEND_UNWRAP_REF, VAR|CV, UNUSED)
```

Comment:
> This RFC proposes to only perform the reference unwrapping for simple variables of the form $value (“CV”), but not for complex variables.

nit: If that's planned, this no longer needs to support VAR?

```suggestion
ZEND_VM_HANDLER(203, ZEND_UNWRAP_REF, CV, UNUSED)
```